.github/workflows/ci.yaml

```diff
@@ -0,0 +1,77 @@
+name: ci
+on: push
+jobs:
+  build27:
+    runs-on: ubuntu-latest
+    env:
+      NOSE_SHOW_SKIPPED: 1
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up python 2.7 with tox environment py27-{test,cov} on ubuntu-latest
+        uses: actions/setup-python@v2
+        with:
+          python-version: 2.7
+      - uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install base dependencies
+        run: |
+          python -m pip install --upgrade pip
+          python -m pip install tox sphinx
+          python -m pip install codecov || true
+      - name: Test with tox
+        run: |
+          sudo ln -s /dev/null /dev/raw1394
+          tox -e py27-test
+          tox -e py27-cov
+          codecov || true
+  build3x:
```

Comment:
Just for clarity, maybe it would be slightly easier to read if these were, like, "build-py3x" and "build-py27"?

Reply:
Sure -- no skin off my back.